test/inductor/test_torchinductor.py

```diff
@@ -7513,6 +7513,37 @@ def fn(in_ptr0, in_ptr1, in_ptr2):
             ),
         )
 
+    def test_indirect_broadcast_embedding(self):
+        B, T, D, V = (8, 2048, 4096, 2048)
+        op = nn.Embedding(V, D).to(self.device).to(torch.float32)
+        shared_weight = op.weight.data
+        tmp_op = nn.Embedding(V, D).to(self.device).to(torch.float32)
+        tmp_op.weight.data.copy_(shared_weight)
```

Comment:
why you need to make a copy of the embedding op?

Reply:
If compile the previous `op`, the inductor_ms is always 0.09 for both before and after this fix. I don't know why it happens and can't reproduce it in an [independent script](https://github.com/FindHao/ml_scripts/blob/main/inductor/cases/embedding/run_embedding.py). any ideas?

Reply:
self.common will cast the model to low precision. You can pass in check_lowp=False and make the test cleaner. But not really a big deal

Reply:
`check_lowp=False` works! fixed in https://github.com/pytorch/pytorch/pull/145992/commits/59e131647fe1bdc9a3c53b8643421629a44ff8fc